composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "silverstripe/recipe-authoring-tools",
+    "type": "silverstripe-recipe",
+    "description": "Extra tools for CMS authoring in SilverStripe",
+    "homepage": "https://www.silverstripe.org",
+    "license": "BSD-3-Clause",
+    "require": {
+        "silverstripe/recipe-plugin": "^1",
+        "silverstripe/recipe-cms": "1.1.x-dev",
+        "silverstripe/documentconverter": "2.x-dev",
+        "silverstripe/iframe": "2.0.x-dev",
+        "silverstripe/spellcheck": "2.x-dev",
+        "silverstripe/tagfield": "2.x-dev",
+        "silverstripe/taxonomy": "2.0.x-dev"
+    },
+    "suggest": {
+        "silverstripe/textextraction": "3.x-dev (not upgraded yet)"
```

Comment:
textextraction should probably be in search recipe, as it's used for providing file content for searching.

Reply:
Cool, it's not ready yet anyway so I'll remove it